client/i18n/locales/english/intro.json

```diff
@@ -5013,6 +4980,10 @@
       "exam-certified-full-stack-developer": {
         "title": "Certified Full Stack Developer Exam",
         "intro": ["Pass this exam to become a Certified Full Stack Developer."]
+      },
+      "workshop-quincys-job-tips": {
```

Comment:
can you move this to right above "lecture-working-with-specialized-semantic-elements"? same order as in full-stack-developer.json